packages/api/src/routes/medical/dtos/patient-cohort.ts

```
@@ -1,4 +1,5 @@
 import { PatientCohort } from "@metriport/shared/domain/patient-cohort";
+import { PatientCohort } from "@metriport/shared/domain/cohort";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'PatientCohort'. Consider to delete it or rename it.

'PatientCohort' is defined here:


(lint/suspicious/noRedeclare)